client_integration_test.go

```diff
@@ -635,13 +645,14 @@ func TestClientRename(t *testing.T) {
 	defer cmd.Wait()
 	defer sftp.Close()
 
-	f, err := ioutil.TempFile("", "sftptest")
-	if err != nil {
-		t.Fatal(err)
-	}
+	dir, err := ioutil.TempDir("", "sftptest-rename")
+	require.Nil(t, err)
```

Comment:
`require.NoError(t, err)` is better.

Reply:
Fixed. I didn't know about that function. I'm going to use it in my own code now too :)